index.html

```diff
@@ -10,8 +11,40 @@
   <link href="./index.css" rel="stylesheet">
 
 </head>
+
 <body>
-<div class="hero">
-</div>
+  <main>
+
+    <div class="hero">
+      <header>
+        <div class="logo">TESLA</div>
+        <div class="menu"><ul>
+          <li>MODEMODEL L S</li>
+          <li>MODEL 3</li>
+          <li>MODEL X</li>
+          <li>MODEL Y</li>
+          <li>CYBERTRUCK</li>
+          <li>ROADSTER</li>
+          <li>ENERGY</li>
+        </ul></div>
+        <div class="hamb"><i class="fas fa-bars"></i></div>
+      </header>
+      <div class="image-content i-white"><img src="./assets/image0-white.svg" alt="">
+      </div>
+      <div class="image-content i-black"><img src="./assets/image0.svg" alt="">
+      </div>
+      <div>
+        <p class="content">
+          BETTER UTILITY THAN A TRUCK WITH MORE PERFORMANCE THAN A SPORTS CAR</p>
```

Comment:
For content in SCREAMING LETTERS, better work with css `text-transform: uppercase`, otherwise screen reader will read E-V-E-R-Y letter individually.